wbooks/src/app/components/Books/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, FlatList } from 'react-native';
+
+import BookCard from '../BookCard';
+import { BOOKS_DATA } from '../../../constants/mocks';
+import { Book } from '../../interfaces/books';
+import { styles } from '../Books/styles';
+
+const BooksList = () => {
+  const renderItem = ({ item }: { item: Book }) => (
+    <BookCard title={item.title} author={item.author} img={item.img} />
```

Comment:
Did it show you some warning about missing keys? Any ways you should still add it:
```suggestion
    <BookCard key={item.id} title={item.title} author={item.author} img={item.img} />
```